packages/extensions/venia-adobe-data-layer/intercept.js

```diff
@@ -0,0 +1,7 @@
+module.exports = targets => {
+    const { talons } = targets.of('@magento/peregrine');
```

Comment:
I am assuming this is the reason why you have peregrine as a peer dep, but here were are searching for peregrine targets not importing them. Is that accurate?

Reply:
I assume it's good practice to list anything you expect to be present as a peer dependency. This package won't work if `@magento/peregrine` isn't present, but it also doesn't want to be the one to import it. Is that not a peer dependency?

Open to changing it if my assumptions aren't correct here.

Reply:
You are right about the premise. This extension is not useful if peregrine isn't part of the main package.

But I am not sure if adding it as part of peer dep, for that reason, is justifiable. On top of it, versioning is involved, and if the base app uses a diff version of peregrine this package might complain about a missing version in the console causing more doubts.